joomla/joomla-cms
# [4.0] Highlighter JS improvements

Pull Request for Issue # .

### Summary of Changes

This PR removes the inline JS for the smart search highlighter and also removes 90% of the jQuery.

There's still a couple more lines of jQuery left to rewrite, however I wasn't sure how to test that specific function.

### Testing Instructions

1. Create a smart search module and assign it to a page.
2. Type a word in the input field and press Enter.
3. Ensure the words on the results page are highlighted
